scripts/install-windows-prereqs.ps1

```diff
@@ -449,26 +490,27 @@ function Install-DCAPDrivers {
     $TEMP_NUGET_DIR = "$PACKAGES_DIRECTORY\Azure_DCAP_Client_nupkg"
```

Comment:
Probably useful to add a comment to point out that the ordering of nuget installs is important to preserve here until Intel fixes the package versioning.